PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,10 +3,11 @@
 What does this PR do?
 
 #### Any relevant tickets
+Please use the phrase 'Closes' if it will fully complete the ticket.
 
 #### How has this been tested?
-- [ ] Unit tests
-- [ ] Integration tests
+Please indicate the need for this change to be tested. If not please justify, if so then at what levels. **Reviewers** ensure that these test requirements are also reviewed.
+
+- [ ] Unit / Integration tests
 - [ ] Browser tests
-- [ ] Review
-- [ ] Styleguide
```

Comment:
Yesterday I think we spoke about leaving "visually" and "styleguide" in the checklist, no?